templates/taxi/car_detail.html

```diff
@@ -12,7 +12,18 @@ <h1>
     </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  <h1>Drivers
+    {% if request.user in car.drivers.all %}
+      <a href="{% url 'taxi:driver-car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
+        Delete me from this car
+      </a>
+    {% else %}
+
+      <a href="{% url 'taxi:driver-car-assign' pk=car.id %}" class="btn btn-secondary link-to-page">
+        Assign me to this car
+      </a>
+    {% endif %}
+  </h1>
```

Comment:
as I said, it could be one function with different approach. I think it's not obligatory but consider this information.